src/clojure_mcp/tools/unified_read_file/tool.clj

```diff
@@ -36,7 +36,7 @@
   [file-path]
   (when file-path
     (let [extension (last (str/split file-path #"\."))]
-      (contains? #{"clj" "cljc" "cljs"} extension))))
+      (contains? #{"clj" "cljc" "cljs" "edn" "bb"} extension))))
```

Comment:
So I guess I misnamed this function... It should be called collapsible-clojure-file

Right now we aren't collapsing .edn files so we should leave "edn" out of this list.